server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -6458,4 +6423,4 @@ private boolean checkStatusOfVolumeSnapshots(long vmId, Volume.Type type) {
         }
         return false;
     }
-}
+}
```

Comment:
it is hard to find functional changes in this file. the new 2380-2424 are it, are they?

Reply:
Yeas, it is.

That is why I am not doing like this anymore. I am always splitting up the PR into multiple commits. It was one of your suggestions some time ago, which I liked. However, I created this PR before that.

The changes are basically at lines 2380-2424.